public/cards/guesses.js

```diff
@@ -574,7 +574,7 @@ module.exports = [
 		ranking: [],
 	}, {
 		category: "guess",
-		question: "Wieviel Bytes hat 1Gb?",
+		question: "Wieviel Bytes hat 1GB?",
```

Comment:
```suggestion
		question: "Wieviel Bytes hat 1 Gigabyte?",
```
please be more specific here, people will claim the abbreviation confused the shit out of the

Reply:
Trivia knowledge is underrated